src/Client.js

```diff
@@ -0,0 +1,110 @@
+import Api from './core/Api';
+import SocketHandler from './core/SocketHandler';
+import { Event } from './core/ChainEvent';
+import { find } from 'lodash';
```

Comment:
you really don't need to import all of `lodash` just to use `find` which is a native array method already.